### PR TITLE
Added insert comments and tasks, fixed logic

### PR DESCRIPTION
**Pull request by EduarfF**

### CODE REVIEW

**Comment by makaroshyna** [comment]
Fix your oull request name

File: src/main/resources/db/changelog/changes/13-insert-tasks.yaml [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+                  value: "HIGH"
+              - column:
+                  name: status
+                  value: "IN_PROGRESS"
+              - column:
+                  name: due_date
+                  valueDate: "2024-06-05"
+              - column:
+                  name: project_id
+                  value: 1
```
Comment (makaroshyna) [inline src/main/resources/db/changelog/changes/13-insert-tasks.yaml:26]: It is better to do valueComputed here, ID could change.
```suggestion
                  valueComputed: (SELECT id FROM projects WHERE name = 'some name')
```

File: src/main/resources/db/changelog/changes/13-insert-tasks.yaml [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+              - column:
+                  name: user_id
+                  value: 1
```
Comment (makaroshyna) [inline src/main/resources/db/changelog/changes/13-insert-tasks.yaml:29]: Here too. And check the other insert too.

File: src/main/resources/db/changelog/changes/14-insert-comments.yaml
Comment (makaroshyna) [inline src/main/resources/db/changelog/changes/14-insert-comments.yaml:1]: Add valueComputed here, don't just set ID values.

**Review by makaroshyna — CHANGES_REQUESTED** [review]

**Review by Bohadan-Markatov — APPROVED** [review]
gj

**Review by makaroshyna — APPROVED** [review]

**Pull request merged**